.gas-snapshot

```diff
@@ -19,42 +19,48 @@ Certificate_supportsInterface:test() (gas: 5159)
-Checkout_buyingFromOneRemoval:test() (gas: 726440)
-Checkout_buyingFromOneRemoval_byApproval:test() (gas: 684688)
-Checkout_buyingFromSingleSupplierWithCustomFee:test() (gas: 547923)
-Checkout_buyingFromTenRemovals:test() (gas: 1976973)
-Checkout_buyingFromTenRemovals_singleSupplier:test() (gas: 1965977)
-Checkout_buyingFromTenRemovals_singleSupplier_byApproval:test() (gas: 1963977)
-Checkout_buyingFromTenRemovals_singleSupplier_withoutFee:test() (gas: 1708460)
-Checkout_buyingFromTenRemovals_withoutFee:test() (gas: 1719464)
-Checkout_buyingFromTenSuppliers:test() (gas: 2855260)
-Checkout_buyingWithAlternateERC20:test() (gas: 544721)
-Checkout_buyingWithAlternateERC20_floatingPointPriceMultiple:test() (gas: 544721)
-Checkout_buyingWithCustomFee:test() (gas: 547446)
-Checkout_swapWithDifferentPermitSignerAndMsgSender:test() (gas: 726890)
+Checkout_buyingFromOneRemoval:test() (gas: 726000)
+Checkout_buyingFromOneRemoval_byApproval:test() (gas: 684424)
+Checkout_buyingFromTenRemovals:test() (gas: 1975477)
+Checkout_buyingFromTenRemovals_singleSupplier:test() (gas: 1964590)
+Checkout_buyingFromTenRemovals_singleSupplier_byApproval:test() (gas: 1962590)
+Checkout_buyingFromTenRemovals_singleSupplier_withoutFee:test() (gas: 1707581)
+Checkout_buyingFromTenRemovals_withoutFee:test() (gas: 1718584)
+Checkout_buyingFromTenSuppliers:test() (gas: 2853764)
+Checkout_buyingWithAlternateERC20:test() (gas: 544547)
+Checkout_buyingWithAlternateERC20_floatingPointPriceMultiple:test() (gas: 544547)
+Checkout_swapFromSupplierWithoutFeeSpecialOrder:test() (gas: 563352)
+Checkout_swapWithDifferentPermitSignerAndMsgSender:test() (gas: 726450)
+Checkout_swapWithoutFeeSpecialOrder:test() (gas: 564859)
 LockedNORILib_availableAmount:test() (gas: 12371)
 LockedNORITest:testBatchGrantCreation() (gas: 705179)
 LockedNORITest:testNormalWithdrawal() (gas: 1102743)
 LockedNORITest:testReentryTokensReceived() (gas: 1102887)
 LockedNORITest:testReentryTokensToSend() (gas: 1104432)
 LockedNORITest:testTokensReceivedReverts() (gas: 69026)
-MarketInvariantTest:invariant_callSummary() (runs: 400, calls: 6000, reverts: 4265)
-MarketInvariantTest:invariant_sumOfPurchaseAmounts() (runs: 400, calls: 6000, reverts: 4265)
-MarketSupplierSelectionNotUsingUpSuppliersLastRemoval:test() (gas: 922133)
-Market_ALLOWLIST_ROLE:test() (gas: 12799)
-Market_USDC_swap_respects_decimal_mismatch:test() (gas: 716909)
-Market_USDC_swap_withholds_restricted_nori:test() (gas: 892028)
+MarketGasExplorer:test_0_bestPossibleCaseOneRemoval() (gas: 29769122)
+MarketGasExplorer:test_1_mostRealisticSeveralSuppliersModerateAmounts() (gas: 49820714)
+MarketGasExplorer:test_2_uniqueSuppliersBulkAmounts() (gas: 78231132)
+MarketGasExplorer:test_3_uniqueSuppliersVariableAmounts() (gas: 1808717194)
+MarketGasExplorer:test_4_singleSupplierGranularAmounts() (gas: 3985218350)
+MarketGasExplorer:test_5_uniqueSuppliersGranularAmounts() (gas: 6532333334)
```

Comment:
I merged up with master and I thought these "MarketGasExplorer" tests would be omitted after those changes so i'm not sure why they're appearing in this snapshot